src/View/Circuits/chooseCircuits.html.twig

```diff
@@ -0,0 +1,37 @@
+{% extends "layout.html.twig" %}
+
+{% block stylesheet %}
+	<link rel="stylesheet" href="/../assets/css/circuits.css">
+{% endblock %}
+
+{% block title %}
+	{{parent()}}
+	| Circuits
+{% endblock %}
+
+{% block content %}
+	<section>
+		<div class="container-fluid bg-secondary py-5 vh-100">
+			<div class="d-flex justify-content-center p-4">
+				<h1>Nos Circuits</h1>
+			</div>
+			<div class="px-5 py-3">
+				<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis optio adipisci ratione, nulla non ullam aut voluptatum odit corporis dolor ducimus. Ratione recusandae maxime laboriosam, non libero, quae iure numquam tempore ad earum necessitatibus illum tempora excepturi mollitia sint fuga?</p>
```

Comment:
Evitez le lorem car c'est un texte non administrable et il faut revenir dessus dans un fix. Si besoin d'un texte "métier" il faut le demander au client.
Pour cette page, mettre qq chose comme "L'association vous propose de découvrir le gâtinais au travers de différents circuits, durant lesquels vous aurez l'occasion d'explorer la faune et la flore de notre région".

Reply:
C'est noté Merci , prochain fix et rdv client a prévoir